xdsl/transforms/transform_interpreter.py

```diff
@@ -0,0 +1,43 @@
+"""Transform interpreter."""
+
+from dataclasses import dataclass
+
+from xdsl.dialects import builtin, transform
+from xdsl.interpreter import Interpreter
+from xdsl.interpreters.transform import TransformFunctions
+from xdsl.passes import Context, ModulePass
+from xdsl.utils.exceptions import PassFailedException
+
+
+@dataclass(frozen=True)
+class TransformInterpreterPass(ModulePass):
+    """Transform dialect interpreter"""
+
+    # TODO: Add the rest of options to the transform interpreter
+    # and add semantics for all of the other transform operations
```

Comment:
let's remove this comment, we avoid TODOs and track these with issues instead

Reply:
https://github.com/xdslproject/xdsl/pull/4289/commits/1b8391835ca362b8f2c2212cc1aaf5b613830c63